src/main/java/io/github/dpsoft/ap/ProfilerExecutor.java

```diff
@@ -79,14 +84,17 @@ private Try<Void> toFirefoxProfiler(OutputStream out) {
         });
     }
 
-    private Try<Void> toFlame(Output type, OutputStream out) {
+    private Try<Void> toFlame(Output type, Command command, OutputStream out) {
```

Comment:
I think we can remove `Output type` param and just use the `Command command` and call `command.output`